_components/pages/History.tsx

```diff
@@ -0,0 +1,213 @@
+      <section className="py-26 text-center">
+        <div className="col-lg-5 mx-auto text-center">
+          <div className="d-flex flex-column-reverse">
+            <h1 className="mb-0">Provide a Better Alternative to Bitcoin</h1>
+            <h6 className="eyebrow mb-3">XRPL's Origin</h6>
+          </div>
+        </div>
+      </section>
+
+      <section className="container-new py-26">
+        <div className="col-lg-6 offset-lg-3">
+          <h4 className="longform mb-10">
+            In 2011, three engineers—David Schwartz, Jed McCaleb, and Arthur Britto—began
+            developing the XRP Ledger (XRPL). Fascinated by Bitcoin, they set out to
+            create a better version that improved upon its limitations—with the goal of
+            creating a digital asset that was more sustainable and built specifically for
+            payments.
+          </h4>
+          <p className="mb-6">
+            The XRP Ledger first launched in June 2012. Shortly thereafter, they were
+            joined by Chris Larsen, and the group started the Company NewCoin in September
+            2012 (quickly renamed OpenCoin and now named Ripple).
+          </p>
+          <p className="mb-0">
+            The XRPL founders gifted 80 billion XRP, the platform’s native currency, to
+            the company. Ripple has since put the majority in escrow.
+          </p>
+        </div>
+      </section>
+
+      <div className="position-relative d-none-sm">
+        <img src={prefix + "/img/backgrounds/history-purple.svg"} id="history-purple" />
+      </div>
+
+      <div className="container-new marketing-wrapper">
+        <section className="row mb-60">
+          <div className="timeline">
+            <div className="timeline-block mb-20-sm mb-10-until-sm">
+              <div className="timeline-dot"></div>
+              <div className="timeline-content text-sm-left text-md-right">
+                <div className="d-flex flex-column-reverse">
+                  <h2 className="mb-6 h5 h2-sm">2011 XRP Ledger Development</h2>
+                  <h6 className="h1 mb-3">2011</h6>
+                </div>
+                <p>
+                  In early 2011, three developers—David Schwartz, Jed McCaleb, and Arthur
+                  Britto—were fascinated with Bitcoin but observed the waste inherent in
+                  mining. They sought to create a more sustainable system for sending
+                  value (an idea outlined in a{" "}
+                  <a
+                    href="https://bitcointalk.org/index.php?topic=10193.0"
+                    target="_blank"
+                  >
+                    May 2011 forum post: “Bitcoin without mining”
+                  </a>
+                  ).
+                </p>
+                <a
+                  className="btn btn-primary read-more mt-10"
+                  href="#"
+                  data-target="section-1"
+                >
+                  Read More
+                </a>
```

Comment:
For some reason the "Read more" buttons scroll me back to the top of the page. I thought maybe it was something to do with JSX needing hyphenated attributes like `data-target` to be camelCase (`dataTarget`) instead, but that alone doesn't seem to have fixed it. Maybe there's also some custom JS that's normally part of this page?

Reply:
I think this needs a react refactor to show the hidden paragraph, I started writing the refactor.

Reply:
refactor completed read more should work now